### PR TITLE
AOM-131: Improve UI Look

### PR DESCRIPTION
**Pull request by patlub**
## JIRA TICKET NAME:

[AOM-131: Improve UI Look](https://issues.openmrs.org/browse/AOM-131)

### SUMMARY:

Improve UI by removing unnecessary repetitions and in components

### CODE REVIEW

**Comment by patlub** [comment]
screen shot 2018-01-26 at 7 13 32 pm

**Comment by Annettesunday** [comment]
Why did you attach the screenshot on the PR? @patlub

**Comment by malmike** [comment]
Looks good to me

**Comment by Annettesunday** [comment]
Why did you attach the screenshot on the PR? @patlub

**Comment by Annettesunday** [comment]
Looks good to me

**Comment by dkayiwa** [comment]
@patlub can you take a look at the merge conflicts?

**Comment by dkayiwa** [comment]
Is "yarn.lock" part of this commit? Or was it committed accidentally?

File: app/js/components/manageApps/AddAddon.jsx [hunk 1]
```diff
@@ -24,7 +24,7 @@ export default class AddAddon extends Component {
     const { files } = this.props;
     if (files == null || files.length < 1) {
       return (
-        <p>Drag file here to upload or
+        <p>Drag addon here to upload or
```
Comment (dkayiwa) [inline app/js/components/manageApps/AddAddon.jsx]: Should we use Add-on for consistency?

Comment (patlub) [reply app/js/components/manageApps/AddAddon.jsx]: Yes. Let me fix it.

**Pull request merged**